pkg/config/config.go

```diff
@@ -195,6 +196,21 @@ var defaultServerConfig = &ServerConfig{
 		MaxMemoryConsumption:   16 * 1024 * 1024 * 1024, // 16GB
 		NumWorkerPoolGoroutine: 16,
 		SortDir:                DefaultSortDir,
+
```

Comment:
Maybe we can add some comments here (or in the doc) to prevent users from modifying these important leveldb configuration items.